.idea/codeStyles/Project.xml

```diff
@@ -1,8 +1,8 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <option name="INSERT_INNER_CLASS_IMPORTS" value="true" />
-    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
```

Comment:
When I changed settings to prevent wildcard imports, "CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" was the field under Java that changed. I unthinkingly changed them in this top-level area as well, but I probably shouldn't have because it seems like I only needed to change them for Java and Kotlin (which confusingly seems to be affected by "JetCodeStyleSettings"). I'll change these two lines back to the way they were before.